src/dcd_mapping/main.py

```diff
@@ -223,7 +233,8 @@ async def map_scoreset(
         )
         _emit_info(f"Score set mapping output saved to: {final_output}.", silent)
         return
-    if vrs_results is None:
+    # TODO this should be if all values in dict are none. or might not need this at all.
+    if vrs_results is None or len(vrs_results) == 0:
```

Comment:
Since we set `vrs_results` to an empty dictionary it can never be None, so this check could just be
```suggestion
    if not vrs_results:
```